### DISCUSSION TITLE
No corre un programa por el kernel

### DISCUSSION THREAD

**Discussion by AlanAcosta17** [LaunchX-InnovaccionVirtual/CursoIntroPython#86] [Q&A]
Buenas tardes, ayer estuve checando la documentación y hoy al abrir de nuevo el VS ya no me dejaba correr los programas. Me decía que si por la extension de Jupyter o si python y ahora me aparece que el kernel. ¿Me podrían ayudar?
![image](https://user-images.githubusercontent.com/99235417/153500112-992c783a-df90-4f3d-9680-03d4e340ca4f.png)

**Comment by MarcosSilvaG** [comment]
¿intentaste seleccionar un kernel diferente?

**Reply by AlanAcosta17** [reply]
Eso se hace instalando otras versiones de python verdad?

**Reply by JosueVerAlar** [reply]
Revisa si tienes actualizada la extensión de Jypiter y la de python dentro de VSCode, puede que ese sea el problema, saludos!

**Comment by Dfredude** [comment]
Corre en el CLI (bash) si estás en UNIX o Linux:
`sudo apt install python3-pip`
`pip3 install ipykernel
`
